semeio/jobs/config_jobs/GENDATA_RFT

```diff
@@ -0,0 +1,12 @@
+STDOUT        GENDATA_RFT.stdout
+STDERR        GENDATA_RFT.stderr
+EXECUTABLE    ../scripts/gendata_rft.py
+ARGLIST       <ECL_BASE> <PATH_TO_TRAJECTORY_FILES> <WELL_AND_TIME_FILE> <ZONEMAP>
```

Comment:
The argument should use the corresponding short-name  in the args-list, i.e. `-e <ECL_BASE> .. -z <ZONEMAP>`

Reply:
Thanks, fixed!